python/morsePersistence.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'Plane'
+plane1 = Plane()
+plane1.XResolution = 300
+plane1.YResolution = 300
+
+# create a new 'Tetrahedralize'
+tetrahedralize2 = Tetrahedralize(Input=plane1)
+
+# create a new 'Random Attributes'
+randomAttributes1 = RandomAttributes(Input=tetrahedralize2)
+randomAttributes1.DataType = 'Float'
+randomAttributes1.ComponentRange = [0.0, 1.0]
+randomAttributes1.GeneratePointScalars = 1
+randomAttributes1.GenerateCellVectors = 0
+
+# create a new 'TTK ScalarFieldSmoother'
+tTKScalarFieldSmoother1 = TTKScalarFieldSmoother(Input=randomAttributes1)
+tTKScalarFieldSmoother1.ScalarField = ['POINTS', 'RandomPointScalars']
+tTKScalarFieldSmoother1.IterationNumber = 6
+tTKScalarFieldSmoother1.MaskField = ['POINTS', '']
```

Comment:
I don't believe this line is necessary. 
Thanks for double checking (e.g. with `diff` or `paraview`) that the outputs of the script are unchanged after the edit.